### PR TITLE
DOC-9104: Added information on smart batching and batch size

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

File: modules/learn/pages/clusters-and-availability/rebalance.adoc [hunk 1]
```diff
@@ -164,6 +164,32 @@ If more index replicas exist than can be handled by the number of existing nodes
 During rebalance, no index node is removed until index-building has completed on alternative nodes.
 This ensures uninterrupted access to indexes.
 
+==== Smart Batching
+
+When rebalance occurs, indexes and relocated and rebuilt in _batches_, so as to place a limit on the amount of work that is attempted concurrently -- the default batch size being `3`.
```
Comment (cherkauer-couchbase) [inline modules/learn/pages/clusters-and-availability/rebalance.adoc:169]: "indexes and relocated" --> "indexes are relocated"

File: modules/learn/pages/clusters-and-availability/rebalance.adoc [hunk 2]
```diff
@@ -164,6 +164,32 @@ If more index replicas exist than can be handled by the number of existing nodes
...
+
+In Couchbase Server Version 7.1 and after, _smart batching_ is used; so as to reduce further the time and resources required to move index metadata, and to rebuild indexes at their new locations during rebalance.
+Specifically, smart batching:
+
+* Increases rebalance performance by increasing pipeline parallelism.
```
Comment (cherkauer-couchbase) [inline modules/learn/pages/clusters-and-availability/rebalance.adoc:174]: Add: "through starting the next batch while the prior batch is only partially completed."

File: modules/rest-api/pages/rest-modify-index-batch-size.adoc [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+== HTTP Method and URI
+
+----
+POST /settings
+----
+
+== Description
+
+Establishes the _batch size_, which is used for index rebuilding and metadata transfer, during rebalance.
```
Comment (cherkauer-couchbase) [inline modules/rest-api/pages/rest-modify-index-batch-size.adoc:16]: Add: "This is the maximum number of indexes that will be moved concurrently in the cluster."

File: modules/learn/pages/clusters-and-availability/rebalance.adoc (hunk 1 above)
Comment (tonyjhillman) [reply modules/learn/pages/clusters-and-availability/rebalance.adoc:169]: Thanks, have changed.

File: modules/learn/pages/clusters-and-availability/rebalance.adoc (hunk 2 above)
Comment (tonyjhillman) [reply modules/learn/pages/clusters-and-availability/rebalance.adoc:174]: Have added.

File: modules/rest-api/pages/rest-modify-index-batch-size.adoc (hunk 3 above)
Comment (tonyjhillman) [reply modules/rest-api/pages/rest-modify-index-batch-size.adoc:16]: Have added.

**Comment by tonyjhillman** [comment]
Thanks!

**Comment by tonyjhillman** [comment]
Added changes on https://github.com/couchbase/docs-server/pull/2572/

**Pull request merged**